### ISSUE TITLE
flowjo_to_dataset: Error in datarange[2, ] : incorrect number of dimensions

### ISSUE THREAD

**Issue by melemieux** [RGLab/CytoML#72]
Hi!

First off, thanks for this great suite of tools! Until this morning all was well but then I decided to install the latest CytoML from trunk (and openCyto, flowCore, etc. etc.). Now flowjo_to_dataset will only load one file and then throws "Error in datarange[2, ] : incorrect number of dimensions" and stops. The last install I had done was end of August this year.

After fiddling a bit, I removed all the packages and reinstalled everything fresh from trunk in the order indicated:
library(devtools)
install_github("RGLab/RProtoBufLib",ref="trunk")
install_github("RGLab/cytolib",ref="trunk")
install_github("RGLab/flowCore",ref="trunk")
install_github("RGLab/flowViz",ref="trunk")
install_github("RGLab/ncdfFlow",ref="trunk")
install_github("RGLab/flowWorkspace",ref="trunk")
install_github("RGLab/flowStats",ref="trunk")
install_github("RGLab/openCyto",ref="trunk")
install_github("RGLab/ggcyto",ref="trunk")
install_github("RGLab/CytoML",ref="trunk")

Details of the error in attached file.

Thanks for any help. I'm stumped.
Madeleine
[flowjo2gs_dimensions.txt](https://github.com/RGLab/CytoML/files/3786169/flowjo2gs_dimensions.txt)

**Comment by jacobpwagner** [comment]
Hi @melemieux , thanks for bringing this to my attention. I had a seen a similar error with another workspace and this helped me run down the issue. It should hopefully be fixed after RGLab/flowWorkspace@a1f042069ef0dc61d52d988525c32b016b4f0455, so please try again reinstalling `flowWorkspace` using:
```
install_github("RGLab/flowWorkspace",ref="trunk", force = TRUE)`
```
and then trying again. If that doesn't work, please let me know and I'll look for other potential causes.

**Comment by melemieux** [comment]
Beauty! Works fine again.

Thanks for the speedy response.
Madeleine
